packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -142,13 +142,14 @@ def test_octarine_srv(cluster, timeout=30):
 
 def test_pkgpanda_api(cluster):
 
-    def get_and_validate_package_ids(node, uri):
-        r = cluster.get(node=node, path=uri)
+    def get_and_validate_package_ids(node, path):
+        node_url = cluster.get_node_url(node)
```

Comment:
I thought wrapping a wrapper was bad but having to call cluster twice in one go is a little worse I now feel, so I will give it a shot